src/async-components/views/dialogs/security/CreateSecretStorageDialog.tsx

```diff
@@ -177,26 +176,29 @@ export default class CreateSecretStorageDialog extends React.PureComponent<IProp
         this.fetchBackupInfo();
     }
 
-    private async fetchBackupInfo(): Promise<{ backupInfo?: IKeyBackupInfo; backupSigStatus?: TrustInfo }> {
+    private async fetchBackupInfo(): Promise<{ backupInfo?: IKeyBackupInfo; backupEnabled?: boolean }> {
         try {
             const cli = MatrixClientPeg.safeGet();
             const backupInfo = await cli.getKeyBackupVersion();
-            const backupSigStatus =
+            const enabled =
                 // we may not have started crypto yet, in which case we definitely don't trust the backup
-                backupInfo && cli.isCryptoEnabled() ? await cli.isKeyBackupTrusted(backupInfo) : null;
+                backupInfo && cli.getCrypto() != null
+                    ? (await cli.getCrypto()!.isKeyBackupTrusted(backupInfo)).trusted
+                    : false;
 
+            const backupEnabled = enabled;
             const { forceReset } = this.props;
             const phase = backupInfo && !forceReset ? Phase.Migrate : Phase.ChooseKeyPassphrase;
 
             this.setState({
                 phase,
                 backupInfo,
-                backupSigStatus,
+                backupEnabled,
             });
 
             return {
                 backupInfo: backupInfo ?? undefined,
-                backupSigStatus: backupSigStatus ?? undefined,
+                backupEnabled: backupEnabled ?? undefined,
```

Comment:
can this ever return `undefined`? (ie, can ` backupEnabled` ever be null or undefined here?)